src/components/common/needsShutdown.jsx

```diff
@@ -159,7 +159,10 @@ export const VmNeedsShutdown = ({ vm }) => {
             position="bottom"
             hasAutoWidth
             bodyContent={body}>
-            <Label className="resource-state-text" href="javascript:null" color="blue" id={`vm-${vm.name}-needs-shutdown`} icon={<PendingIcon />}>{_("Changes pending")}</Label>
+            <Label className="resource-state-text" color="blue" id={`vm-${vm.name}-needs-shutdown`}
+                   icon={<PendingIcon />} onClick={() => null}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1274-20231027-124301-1f1884f1-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)